src/Documentation/SidebarMenu/helper.test.js

```diff
@@ -257,6 +257,7 @@ describe('SidebarMenu/helper', () => {
       )
     })
 
+    // eslint-disable-next-line max-len
     it("Throws error if item has source: false and doesn't have children", () => {
```

Comment:
> Migrating from https://github.com/iterative/dvc.org/pull/715#pullrequestreview-308263027:

I think we could've just shortened this string to something like `"Throws error  if source:false in leaf item"`.

Reply:
@jorgeorpinel I think @shcheklein can answer that better.

Reply:
OK